helper/common/common.go

```diff
@@ -79,6 +79,24 @@ func ConvertUnmarshalledUint(x interface{}) (uint64, error) {
 	}
 }
 
+// ParseUint64orHex parses the given uint64 hex string into the number.
+// It can parse the string with 0x prefix as well.
+func ParseUint64orHex(val *string) (uint64, error) {
```

Comment:
Don't we have the same function in `types` package?

Reply:
Yes, we do have it, but as Roman explained, the aim is to remove that one in some of the following PRs and use this one instead.

Reply:
Ah, OK, didn't see that he already explained that somewhere.